app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -113,6 +114,11 @@ sealed class Event {
     object PrivateBrowsingPinnedShortcutPrivateTab : Event()
     object PrivateBrowsingStaticShortcutTab : Event()
     object PrivateBrowsingStaticShortcutPrivateTab : Event()
+    object TrackingProtectionTrackerList : Event()
```

Comment:
I don't see this event been use in another place, is it intentionally?

Reply:
Oops updating to use it now :)

Reply:
:)